### PR TITLE
SW-2558 Change font size on withdrawals details pages

### PR DESCRIPTION
**Pull request by karthikbtf**
- match figma design
- also updated a padding on the map (unrelated change)

Terraware App 2022-12-20 07-27-21

Terraware App 2022-12-20 07-27-03

### CODE REVIEW

File: src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx [hunk 1]
```diff
@@ -174,7 +174,7 @@ export default function NurseryWithdrawalsDetails({
             justifyContent='space-between'
             alignItems='center'
           >
-            <Typography color={theme.palette.TwClrTxt} fontSize='20px' fontWeight={600} fontStyle='italic'>
+            <Typography color={theme.palette.ClrTextFill} fontSize='24px' lineHeight='32px' fontWeight={600}>
```
Comment (charlie-tf) [inline src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:177]: This is a token from the old theme. Do we want to use it here?

Comment (karthikbtf) [reply src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:177]: We don't have this color in the new theme. I added a comment in figma asking the new theme to be updated. We have a few other such usages in our codebase.

Comment (karthikbtf) [reply src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:177]: see https://www.figma.com/file/yRSfer1xtWKAJr4IETPml1?node-id=2074:504340#330623402

Comment (charlie-tf) [reply src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:177]: Thanks!

**Review by charlie-tf — APPROVED** [review]

File: src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:177]: I will create a ticket to track removal of old theme.

**Pull request merged**